packages/@uppy/companion/src/server/controllers/url.js

```diff
@@ -93,7 +93,6 @@ const meta = async (req, res) => {
     return res.json(urlMeta)
   } catch (err) {
     logger.error(err, 'controller.url.meta.error', req.id)
-    // @todo send more meaningful error message and status code to client if possible
```

Comment:
Do we give up on sending a more meaningful error message?

Reply:
We don't want to expose information to the client: https://github.com/transloadit/uppy/pull/3907#discussion_r929940341